### ISSUE TITLE
3F creeks with missing data automatically get assigned target/green

### ISSUE THREAD

**Issue by fawda123** [tbep-tech/tbeptools#16]
An example of geometric means for JEI MC02 and wbid 1816:
```
   id wbid  JEI class Creek_Length_m year CHLAC COLOR COND DO DOSAT NO23 ORGN SALIN TKN TN TP TSS TURB
1 375 1816 MC02    3F       23783.52   NA    NA    NA   NA NA    NA   NA   NA    NA  NA NA NA  NA   NA
```
And the final score:
```
     id wbid  JEI   class target caution investigate   act score 
  <int> <chr> <chr> <chr>  <int>   <int>       <int> <int> <chr> 
1   375 1816  MC02  3F         1      NA          NA    NA Target
```

This is done here https://github.com/tbep-tech/tbeptools/blob/853897fb40f74bb5612ab8e3c5fcf525e453f0b3/R/anlz_tdlcrk.R#L74

Output from the tbeptools matches that from MW, so need to verify that this was on purpose.

**Comment by mikewessel** [comment]
Marcus - you are correct in identifying this as an issue. 3F WBIDs with no
data should not be assigned a target level. In addition, to clarify, 3F
 wbids do not get  a caution score either so the categories are red,
orange, green or no data. Hope this helps and good catch.  I hope to get to
the vignette this week. Mike

**Comment by fawda123** [comment]
Okay, got it.  Here are the score tallies currently. Note that 3F and "No Data" is at zero and there are no 3F creeks scored as "Caution": 
```
     score
class Act Caution Investigate No Data Target
   1    0       0           0       0      1
   2    0       0           0      34      4
   3F   3       0           4       0     64
   3M  10       4           5      75     24
```
And here are the tallies after fixing the scores for 3F creeks with no data.
```
     score
class Act Caution Investigate No Data Target
   1    0       0           0       1      0
   2    0       0           0      34      4
   3F   3       0           4      38     26
   3M  10       4           5      75     24
```

Fix is here ac784b5f4ac139ded8bfa382a177684c4dfaac35
